stdlib/public/SDK/Foundation/PlistEncoder.swift

```diff
@@ -265,8 +265,12 @@ fileprivate struct _PlistKeyedEncodingContainer<K : CodingKey> : KeyedEncodingCo
     }
 
     public mutating func nestedContainer<NestedKey>(keyedBy keyType: NestedKey.Type, forKey key: Key) -> KeyedEncodingContainer<NestedKey> {
-        let dictionary = NSMutableDictionary()
-        self.container[key.stringValue] = dictionary
+        let containerKey = key.stringValue
+				let existingContainer = self.container[containerKey]
```

Comment:
Nit: something is wrong with the indenting here and two lines below.

Reply:
@xwu thank you for these. I updated the code to fix the wrong indentation. Please have a look again